src/AutoRest.CSharp/Mgmt/Generation/MgmtClientBaseWriter.cs

```diff
@@ -698,8 +702,8 @@ protected virtual void WriteLROResponse(CSharpType lroObjectType, string diagnos
                 {
                     _writer.Append($"{ContextProperty}, ");
```

Comment:
So we don't need to pass `_userAgent` for non-long-running operation?

Reply:
My bad, fixed

Reply:
FYI, I decide not to implement this feature in non-long-running operation, the reason is that for non-long-running operation we initialize the [`_operation`](https://github.com/Azure/autorest.csharp/blob/8436f4e9b38bfb6ca00ac70eaf165e59b09e4175/src/AutoRest.CSharp/Mgmt/Generation/NonLongRunningOperationWriter.cs?_pjax=%23js-repo-pjax-container%3Afirst-of-type%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%3Afirst-of-type%2C%20%5Bdata-pjax-container%5D%3Afirst-of-type#L35) using `OperationOrResponseInternals` or `OperationOrResponseInternals<T>`. As we use response to create this `_operation`, everything is not related with LRO.